virtual-workshop/workshop-materials.md

```diff
@@ -1,12 +1,15 @@
-# Workshop Materials
+---
+title: Workshop Materials
+nav_title: Materials
+---
 
 ### Github Repositories
 
 CCDL training materials are available on Github, in either this repository or our shared [training-modules repository](https://github.com/AlexsLemonade/training-modules).
 
 ### Slides
 
-PDF versions of the slides we present in this workshop can be found in the [slides directory]({{site.repository-url}}/slides) of the `{{site.repository}}` repository, and are also linked to directly from the [schedule](./SCHEDULE.md).
+PDF versions of the slides we present in this workshop can be found in the [slides directory]({{site.repository_url}}/slides) of the `{{site.repository}}` repository, and are also linked to directly from the [schedule](./SCHEDULE.md).
```

Comment:
I think this might actually have to be:
```suggestion
PDF versions of the slides we present in this workshop can be found in the [slides directory]({{site.repository_url}}/tree/master/slides) of the `{{site.repository}}` repository, and are also linked to directly from the [schedule](./SCHEDULE.md).
```

Reply:
And that's probably the case wherever else we use it (I'm not sure if there's anywhere else, though!)